web/.storybook/main.js

```diff
@@ -10,24 +12,35 @@ module.exports = {
       name: "@storybook/addon-postcss",
       options: {
         postcssLoaderOptions: {
-          implementation: require("postcss")
-        }
-      }
-    }
+          implementation: require("postcss"),
+        },
+      },
+    },
   ],
   framework: "@storybook/react",
   core: {
-    builder: "@storybook/builder-webpack5"
+    builder: "@storybook/builder-webpack5",
   },
   features: {
-    storyStoreV7: true
+    storyStoreV7: true,
   },
   webpackFinal: async (config, { configType }) => {
     config.resolve.alias = {
       ...config.resolve.alias,
-      "@src": path.resolve(path.dirname(__dirname), "src/")
+      "@src": path.resolve(path.dirname(__dirname), "src/"),
+    };
+    config.resolve.fallback = {
+      ...config.resolve.fallback,
+      constants: require.resolve("constants-browserify"),
+      zlib: require.resolve("browserify-zlib"),
+      stream: require.resolve("stream-browserify"),
```

Comment:
what are these for?

Reply:
webpack 5 no longer does polyfilling for node core modules, so we have to add fallbacks to the webpack config. 
This modal uses https://github.com/replicatedhq/kots/blob/main/web/src/utilities/utilities.js which imports zlib.